src/Doctrine/Orm/RedirectRoute.php

```diff
@@ -0,0 +1,98 @@
+<?php
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2017 Symfony CMF
```

Comment:
Can you update that to 2018, or are we late on all of our repos?

Reply:
Thats wrong on our end. I edited styleci and suggest we simply remove the years. See #421

Reply:
can you rebase the PR @ilukac ? we removed the years in styleci now.

Reply:
Should be rebased now